Add more context to containerized uninstall (#2679)
ansible/aap-docs: downstream/modules/platform/proc-uninstalling-containerized-aap.adoc

[role="_abstract"]

> When performing a reinstall following an uninstall that preserves the databases, it is necessary to use the previously generated {PlatformNameShort} secret key values to access the preserved databases.

Reviewer: ```suggestion
When performing a reinstall following an uninstall that preserves the databases, you must use the previously generated {PlatformNameShort} secret key values to access the preserved databases.
```
